src/ui/svg/icons/CheckboxMarkChecked.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react'
+import { Path } from 'react-native-svg'
+import styled from 'styled-components/native'
+
+import { AccessibleSvg } from 'ui/svg/AccessibleSvg'
+
+import { AccessibleRectangleIcon } from './types'
+
+const CheckboxMarkSvg: React.FunctionComponent<AccessibleRectangleIcon> = ({
+  width,
+  height,
+  color,
+  accessibilityLabel,
+  testID,
+}) => (
+  <AccessibleSvg
+    width={width}
+    height={height}
+    viewBox="0 0 17 11"
+    accessibilityLabel={accessibilityLabel}
+    testID={testID}>
+    <Path
+      fill={color}
+      d="M6 6.728L2.049 2.776c-.47-.468-1.229-.468-1.698 0-.468.47-.468 1.229 0 1.697l4.8 4.8c.47.47 1.229.47 1.698 0l7.224-7.224c.47-.47.47-1.229 0-1.698-.468-.468-1.228-.468-1.697 0L6 6.728z"
+      transform="translate(-26 -297) translate(22.087 272) translate(0 18) translate(4.8 7.2)"
```

Comment:
c'est dommage de faire faire 4 calculs à chaque fois qu'on affiche ce SVG

on commence avec `M`, en majuscule, donc absolut, je pense qu'on pourrait supprimer le `transform`

```suggestion
```

si on ne peut pas

idéalement, on pourrait changer `d`

-26 + 22.087 + 0 + 4.8 = 0.887
-297 + 272 + 18 + 7.2 = 0.2

si flemme de changer toutes les valeurs, on pourrait au moins faire un seul `translate`

```suggestion
      transform="translate(.887 .2)"
```